### PR TITLE
Add a module to query NEOs (From Poliastro)

### PR DESCRIPTION
**Pull request by shreyasbapat**
Hi!

As discussed on `astroquery` chats some days ago!
https://matrix.to/#/!DdazxkLDTPnBLekbbL:openastronomy.org/$153784692124yzLpI:openastronomy.org

We had a neos module in our library, poliastro which essentially makes it possible to search NEOs by name (also using wildcards), and get their orbits straight from NASA APIs, using some of the defined methods. We can plot the orbit of a NEOs in just 2-3 lines of code. We thought that contributing the NEOs code to astroquery will be a better idea as this will make it even more usable. See the short documentation of NEOs module here: http://docs.poliastro.space/en/latest/user_guide.html#working-with-neos And API reference here: http://docs.poliastro.space/en/latest/safe.html#module-poliastro.neos

I have added some tests for the same as well, kindly let me know the changes required. :)

cc @Juanlu001

### CODE REVIEW

File: astroquery/neos/core.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+            "sstr": name,
+            "orb": "0",
+            "log": "0",
+            "old": "0",
+            "cov": "0",
+            "cad": "0",
+        }
+        SBDB_URL = conf.sbdb_server
+        response = self._request(
+            "GET", SBDB_URL, params=payload, timeout=self.TIMEOUT)
```
Comment (keflavich) [inline astroquery/neos/core.py]: Do you want to (optionally?) enable `cache`ing here?

File: astroquery/neos/core.py [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+        # response = requests.get(SBDB_URL, params=payload)
+        response.raise_for_status()
+        # soup = BeautifulSoup(response.text, "html.parser")
```
Comment (keflavich) [inline astroquery/neos/core.py]:
```suggestion
```

File: astroquery/neos/tests/test_neows.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from ...neos import Neows
+
+from astropy.table import Table
+
+
+def test_observations_from_spk_id():
```
Comment (keflavich) [inline astroquery/neos/tests/test_neows.py]: These are remote-data tests, right?  They need to be tagged with `@remote_data` if so.

File: astroquery/neos/core.py (hunk 1 above)
Comment (shreyasbapat) [reply astroquery/neos/core.py]: Yes, sure. I will do that!

**Comment by bsipocz** [comment]
We already have the jplsbdb module, so I wonder how much of this functionality complementary of what that module can do. My opinion is that querying a subclass of objects from the generic module should not be a new module itself.

cc @mommermi

**Comment by mommermi** [comment]
It seems to me that all the data that can be queried from `NEOWS` originate from `SBDB`, so implementing that service would be redundant. I had a quick look at `Poliastro` and there is a big overlap with what we already have in `astroquery` (`.jplhorizons` and `.jplsbdb`). What would be an interesting new addition is an interface to query orbits from `DASTCOM5`. @shreyasbapat is that something you are planning to implement here?

**Comment by shreyasbapat** [comment]
Thankyou @mommermi , @bsipocz and @keflavich  for prompt replies :)
Yes, I am planning to add another PR for `DASTCOM` after this!

**Comment by shreyasbapat** [comment]
Also, @mommermi , should I fix the issues pointed out by @keflavich or I should perhaps start working on the `DASTCOM5` PR and close this?

**Comment by keflavich** [comment]
@mommermi I noticed you 👍'd the last comment by @shreyasbapat, but he asked an 'or' question.  Could you clarify in text what you thing the right way forward is?  i.e., should we close this PR?

**Comment by mommermi** [comment]
@keflavich I guess it's up to you whether you want to close this PR. To me it sounds like most of the functionality here is already present in jplhorizons or jplsbdb. @shreyasbapat is there anything in this PR that is not yet available in astroquery? (Except for the Dastcom5 module)

**Comment by shreyasbapat** [comment]
It queries [Neows](https://api.nasa.gov/neo/rest/v1/neo/) service provided by JPL :). I don't know until what extent it coincides with SBDB. Yes, a small part of the PR also queries SBDB. If you think it does not have a lot of application, I can happily close it. I am already working on `DASTCOM5` PR as you asked me to do so.

**Comment by bsipocz** [comment]
To me it still looks like it only wraps a convenience service that is actually covered by the current functionalities, so I'm going ahead and close this issue.

Thanks @shreyasbapat for raising this issues.

**Pull request closed without merging**